media/js/base/datalayer-productdownload-init.es6.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrackProductDownload from './datalayer-productdownload.es6';
+
+// Create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+window.Mozilla.TrackProductDownload = TrackProductDownload;
```

Comment:
@alexgibson Is this the right way to make TrackProductDownload available to the thanks page without converting it to ES6?

Reply:
If you want to make a function available to other scripts, without importing the same code in both places, then making it a global is probably the best way to go about it, yes 👍